packages/spindle-ui/src/NavigationTab/UnderlineTab/UnderlineTab.css

```
@@ -0,0 +1,199 @@
+.spui-UnderlineTab {
+  background-color: var(--color-surface-primary);
+}
+
+.spui-UnderlineTab--border {
+  border-bottom: 1px solid var(--color-border-low-emphasis);
+}
+
+.spui-UnderlineTab--scrollable {
+  position: relative;
+}
+
+.spui-UnderlineTab--scrollable .spui-UnderlineTab-container {
+  column-gap: 16px;
+  display: flex;
+  overflow-x: scroll;
+  padding: 0 16px;
+}
+
+.spui-UnderlineTab-button {
+  background-color: transparent;
+  border: none;
+  box-sizing: border-box;
+  flex-shrink: 0;
+  padding: 0 4px;
+  transition: background-color 0.15s ease-in-out;
+}
+
+.spui-UnderlineTab--fixed .spui-UnderlineTab-button {
+  width: 100%;
+}
+
+.spui-UnderlineTab-button:hover {
+  background-color: var(--color-surface-tertiary);
+}
+
+.spui-UnderlineTab-button:focus {
+  border-radius: 4px;
+  outline: 2px solid var(--color-focus-clarity);
+  outline-offset: -2px;
+}
+
+.spui-UnderlineTab-button:not(:focus-visible) {
+  outline: none;
+}
+
+.spui-UnderlineTab-labelWrapper {
+  align-items: center;
+  column-gap: 6px;
+  display: flex;
+  margin: 0 auto;
+  max-width: 100%;
+  padding: 14px 0;
+  position: relative;
+  width: fit-content;
+}
+
+.spui-UnderlineTab-labelWrapper::after {
+  background-color: var(--color-border-accent-primary);
+  border-radius: 3px;
+  bottom: 0;
+  content: '';
+  height: 3px;
+  min-width: 40px;
+  opacity: 0;
+  position: absolute;
+  transition: opacity 0.35s ease-in-out;
+  width: 100%;
+}
+
+/* stylelint-disable-next-line  */
+[aria-selected='true'] .spui-UnderlineTab-labelWrapper::after {
```

```suggestion
.spui-UnderlineTab[aria-selected='true'] .spui-UnderlineTab-labelWrapper::after {
```
だとどうでしょう？`[aria-selected='true']`だと広域すぎるかも。

---

確かにです...！ :sumi: https://github.com/openameba/spindle/pull/909/commits/e2dbdff124be3b263406684ff6761c1f5173945e